db/reqdebug.c

```diff
@@ -52,6 +52,10 @@ void reqerrstr(struct ireq *iq, int rc, char *format, ...)
     if (iq == NULL)
         return;
 
+    /* KNEAD this kludge because of comdb2sc.tsk */
```

Comment:
I'd be a little bit more formal about wording. Format is off as well.

Reply:
done